src/plugins/condition/components/ConditionCollection.vue

```diff
@@ -121,6 +121,9 @@ export default {
         this.conditionCollection = this.domainObject.configuration.conditionCollection;
         this.observeForChanges();
         this.conditionManager = new ConditionManager(this.domainObject, this.openmct);
+        this.conditionManager.on('conditionSetResultUpdated', (data) => {
+            this.$emit('conditionSetResultUpdated', data);
```

Comment:
I would use kebab-case for vue events as the documentation suggests.
https://vuejs.org/v2/guide/components-custom-events.html

Reply:
I had done this before, but Pegah had asked me to change it back to camelCase.